filters/include/pcl/filters/voxel_grid_covariance.h

```diff
@@ -361,14 +361,43 @@ namespace pcl
 
       }
 
+      /** \brief Get the voxels surrounding point p designated by #relative_coordinates.
+       * \note Only voxels containing a sufficient number of points are used.
+       * \param[in] relative_coordinates 3xN matrix that represents relative coordinates of N neighboring voxels with respect to the center voxel
+       * \param[in] reference_point the point to get the leaf structure at
+       * \param[out] neighbors
+       * \return number of neighbors found
+       */
+      int
+      getNeighborhoodAtPoint (const Eigen::Matrix<int, 3, Eigen::Dynamic>& relative_coordinates, const PointT& reference_point, std::vector<LeafConstPtr> &neighbors) const;
+
       /** \brief Get the voxels surrounding point p, not including the voxel containing point p.
```

Comment:
Please state the number of neighbors considered (26?)

Reply:
I updated the comment.